2024/README.md

```diff
@@ -5,3 +5,11 @@
 Improvements:
 - Instead of `os.ReadFile`, could have use `go:embed` to embed those file directly into the static assets
 - Usage `fmt.Sscanf()` to parse the lines into the formatted strings to build up the lists 
+
+## Day 3
+Part Two Solution:
+- Replace all new lines with the "" - Trim Newliens
```

Comment:
Write P2 in the code. Possibly perform the same operations. with string replacements. So it could handle streaming data.